handlers/main.yml

```diff
@@ -14,3 +14,6 @@
     name: supervisor
     state: restarted
   listen: "restart supervisor"
+
+- name: restart netbox
+  supervisorctl: name=netbox state=restarted
```

Comment:
Please adjust to use a YAML dict for this to keep the formatting consistent with the rest of the code.

Reply:
Done.